test/FacadeWrite.test.ts

```diff
@@ -146,10 +154,11 @@ describe('FacadeWrite contract', () => {
     })
     facadeWrite = <FacadeWrite>await FacadeFactory.deploy(deployer.address)
 
-    const revShare = { rTokenDist: bn('2'), rsrDist: bn('3') } // 0.1%
+    revShare = { rTokenDist: bn('2'), rsrDist: bn('3') } // 0.5% for each beneficiary
```

Comment:
Here I'd make sure to use different share amounts for the two beneficiaries

Reply:
Ok will do